samples/PointCloud/src/PointCloudApp.cpp

```diff
@@ -174,14 +176,14 @@ void PointCloudApp::mouseDrag( MouseEvent event )
 {
 	bool middle = event.isMiddleDown()	|| ( event.isMetaDown()		&& event.isLeftDown() );
 	bool right	= event.isRightDown()	|| ( event.isControlDown()	&& event.isLeftDown() );
-	mMayaCam.mouseDrag( event.getPos(), event.isLeftDown() && !middle && !right, middle, right );
+	mCamUi.mouseDrag(event.getPos(), event.isLeftDown() && !middle && !right, middle, right);
```

Comment:
This entire function can be removed. It is now handled automatically by CameraUi.